src/router.ts

```diff
@@ -66,7 +66,7 @@ export class Router {
       if (e instanceof JSONRPCError) {
         return { error: { code: e.code, message: e.message, data: e.data } };
       }
-      return { error: { code: 6969, message: "unknown error" } };
+      return { error: { code: 6969, message: e.message } };
```

Comment:
maybe this could be 

```
{
   error: {
     code: 6969,
     message: "unknown error",
     data: {
       message: e.message
     }
   }
}
```

Reply:
if its not a JSON RPC error, then its an internal error, and we should not leak internal errors to the API user.